### PR TITLE
feat(input/mongodb): support aggregation filters

### PR DESCRIPTION
**Pull request by pcha**
Refs: https://github.com/benthosdev/benthos/issues/1215

### CODE REVIEW

**Comment by pcha** [comment]
The make lint threw some errors not related with my changes, also I ran the integration tests which failed in some components which I didn't touch.

**Comment by pcha** [comment]
Hi, do you think you could see it anytime soon?

File: internal/impl/mongodb/input.go [hunk 1]
```diff
@@ -58,15 +64,15 @@ func newMongoInput(conf *service.ParsedConfig) (service.Input, error) {
...
+	operation, err := conf.FieldString("operation")
 	if err != nil {
 		return nil, err
 	}
-	query, err := queryExecutor.Query(struct{}{})
+	queryExecutor, err := conf.FieldBloblang("query")
 	if err != nil {
 		return nil, err
 	}
-	queryBSON, err := bson.Marshal(query)
```
Comment (Jeffail) [inline internal/impl/mongodb/input.go:69]: Does this bson marshal need to be reintroduced further down?

Comment (Jeffail) [reply internal/impl/mongodb/input.go:69]: 👍 okay sounds good

**Pull request merged**